app/services/cache/ExportItem.scala

```diff
@@ -43,8 +43,13 @@ case class ExportItem(
     fiscalInformation.exists(_.onwardSupplyRelief == FiscalInformation.AllowedFiscalInformationAnswers.yes)
 
   def isCompleted: Boolean =
-    procedureCodes.isDefined && fiscalInformation.isDefined && itemType.isDefined &&
+    procedureCodes.isDefined && isFiscalInformationCompleted && itemType.isDefined &&
       packageInformation.nonEmpty && commodityMeasure.isDefined && additionalInformation.isDefined
+
+  private def isFiscalInformationCompleted: Boolean =
```

Comment:
can we move this to fiscal information object ?

Reply:
this method based on information in fiscal information and additional fiscal information. Decision about completed is connected only from whole item perspective. I think this is better place for this method